packages/common/src/components/logos/Adonis.tsx

```diff
@@ -0,0 +1,14 @@
+import * as React from 'react';
+
+export default ({ width = 32, height = 32, className }) => (
```

Comment:
Shouldn't this one come from the [templates repo](https://github.com/codesandbox/codesandbox-templates)? 🤔

Reply:
I don't know how this happened, deleted again 🗡 

Thank you!